clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileStatus.java

```diff
@@ -0,0 +1,34 @@
+package io.lakefs;
+
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.Path;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+public class LakeFSFileStatus extends FileStatus {
+
+    private String checksum;
+    private String physicalAddress;
+
+    public LakeFSFileStatus(long length, boolean isdir, int block_replication,
+                            long blocksize, long modification_time, Path path,
```

Comment:
nit: Mixed argument styles: snake_case `modification_time` and `block_replication`, uncased `isdir` (and maybe `blocksize`), whackyCapped `physicalAddress`.  Prefer to have just one.

Reply:
sure, thanks!